pkg/env/prod.go

```diff
@@ -91,8 +95,10 @@ func newProd(ctx context.Context, log *logrus.Entry) (*prod, error) {
 
 		fpClientID: os.Getenv("AZURE_FP_CLIENT_ID"),
 
-		clusterGenevaLoggingEnvironment:   os.Getenv("MDSD_ENVIRONMENT"),
+		clusterGenevaLoggingAccount:       os.Getenv("CLUSTER_MDSD_ACCOUNT"),
+		clusterGenevaLoggingNamespace:     os.Getenv("CLUSTER_MDSD_NAMESPACE"),
 		clusterGenevaLoggingConfigVersion: os.Getenv("CLUSTER_MDSD_CONFIG_VERSION"),
+		genevaLoggingEnvironment:          os.Getenv("MDSD_ENVIRONMENT"),
```

Comment:
I think this make sense.

Reply:
>The reason I changed the name of this variable is because even though there are two sets of parameters for this change (cluster mdsd and rp mdsd), the genevaLoggingEnvironment parameter is shared between the two different configurations.

Does this make sense to move this to `pkg/env/core.go`?  Only because looking at the prod struct I expect to see things that are consistent with the cluster and not shared between the rp and the cluster as far as configuration.  I understand that the only instance of this is in this prod struct but explicitly sharing this configuration from the cluster and the RP isn't something I'd expect to see when this struct is used for prod cluster configuration.

Reply:
I think that's a fair point.

Reply:
@mjudeikis @troy0820 I'm going to change the name back.

Reply:
I would rather do that than move anything

Reply:
@troy0820 - I went ahead and changed the names back for everything except the reference to variables in const.go